security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -13,6 +13,49 @@ These release notes track the development of the Compliance Operator in the {pro
 
 For an overview of the Compliance Operator, see xref:../../security/compliance_operator/compliance-operator-understanding.adoc#understanding-compliance-operator[Understanding the Compliance Operator].
 
+[id="compliance-operator-release-notes-0-1-53"]
+== OpenShift Compliance Operator 0.1.53
+
+The following advisory is available for the OpenShift Compliance Operator 0.1.53:
+
+* link:https://access.redhat.com/errata/RHBA-2022:5537[RHBA-2022:5537 - OpenShift Compliance Operator bug fix update]
+
+[id="compliance-operator-0-1-53-bug-fixes"]
+=== Bug fixes
+
+* Previously, the `ocp4-kubelet-enable-streaming-connections` rule contained an incorrect variable comparison, resulting in false positive scan results. Now, the Compliance Operator provides accurate scan results when setting `streamingConnectionIdleTimeout`.  (link:https://bugzilla.redhat.com/show_bug.cgi?id=2069891[*BZ#2069891*])
+
+* Previously, group ownership for `/etc/openvswitch/conf.db` was incorrect on Z architectures, resulting in `ocp4-cis-node-worker-file-groupowner-ovs-conf-db` check failures. Now, the check is marked `NOT-APPLICABLE` on Z architecture systems. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2072597[*BZ#2072597*])
+
+* Previously, the `ocp4-cis-scc-limit-container-allowed-capabilities` rule reported in a `FAIL` state due to incomplete data regarding the SCC rules in the deployment. Now, the result is `MANUAL`, which requires users to review the SCC and ensure they are appropriate for the environment. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2077916[*BZ#2077916*])
+
+* Previously, the following rules failed to account for additional configuration paths for API server and TLS certificates and keys, resulting in reported failures even if the certificates and keys were set properly:
```

Comment:
Should it be "API servers" (plural)? Not sure I'm parsing it right

Reply:
Oh whoops, yes. Good catch! :fishing_pole_and_fish: